rero_ils/modules/entities/local_entities/jsonschemas/local_entities/local_entity_person-v0.0.1.json

```diff
@@ -0,0 +1,144 @@
+{
+  "type": "object",
+  "title": "Person",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "type",
+    "authorized_access_point",
+    "name"
+  ],
+  "propertiesOrder": [
+    "type",
+    "name",
+    "date_of_birth",
+    "date_of_death",
+    "numeration",
+    "qualifier",
+    "fuller_form_of_name",
+    "alternative_names",
+    "gender",
+    "source_catalog",
+    "identifier"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate local entity records against.",
+      "type": "string",
+      "minLength": 9
+    },
+    "type": {
+      "title": "Type",
+      "type": "string",
+      "readOnly": true,
+      "default": "bf:Person",
+      "const": "bf:Person",
+      "form": {
+        "templateOptions": {
+          "wrappers": [
+            "hide"
+          ]
+        }
+      }
+    },
+    "pid": {
+      "title": "Local entity PID",
+      "type": "string"
+    },
+    "deleted": {
+      "title": "Deletion date",
+      "type": "string"
+    },
+    "name": {
+      "title": "Name",
+      "type": "string",
+      "minLength": 1,
+      "form": {
+        "placeholder": "Example: Miller, John"
+      }
+    },
+    "authorized_access_point": {
+      "title": "Authorized access point",
+      "type": "string",
+      "form": {
+        "templateOptions": {
+          "wrappers": [
+            "hide"
+          ]
+        }
+      }
+    },
+    "date_of_birth": {
+      "title": "Birth date",
+      "description": "Record only years.",
+      "type": "string",
+      "minLength": 1,
```

Comment:
No because there might be some litteral (ex. : 17th century, approximatly 1877, active 1679, ...). Maybe we should change "description" into : Mostely, records only the years